log4j-1.2-api/pom.xml

```diff
@@ -43,11 +43,12 @@
       com.sun.jdmk.comm;resolution:=optional,
       <!-- JMS is optional -->
       javax.jms;version="[1.1,3)";resolution:=optional,
-      <!-- Log4j Core is optional -->
-      org.apache.logging.log4j.core.*;resolution:=optional
     </bnd-extra-package-options>
+    <bnd-extra-module-options>
```

Comment:
Don't we also need an entry for `javax.jms:javax.jms-api` too?

Reply:
No, setting the `javax.jms` module to `resolution:=optional` is enough for BND to mark the JPMS module as optional.